# This service requests ...

geometry_msgs/Pose pose 
---
bool success
